setup/install/providers/azure.md

```diff
@@ -52,15 +52,25 @@ RESOURCE_GROUP="Spinnaker"
 az group create --name $RESOURCE_GROUP --location <Insert Location>
 ```
 
-Finally, create a Key Vault (where the vault name is globally unique) and add a default username/password:
+Finally, create a Key Vault (where the vault name is globally unique) and add a default username/ssh public key. This credential is used to provision Azure VM scale set. You can log on to VM instances in the VM scale set with this credential.
```

Comment:
Should up date password on line 19 above to ssh public key as well.

Reply:
Also the credential is used for all VM scale sets by default. Correct? If so, this should be clear here.

Reply:
Good catch. Resolved

Reply:
> Should up date password on line 19 above to ssh public key as well.

Good catch. Resolved

Reply:
> Also the credential is used for all VM scale sets by default. Correct? If so, this should be clear here.

resolved